cli/tests/051_wasm_import.ts

```diff
@@ -0,0 +1,24 @@
+import wasmExports from "./051_wasm_import/simple.wasm";
```

Comment:
Hmmm, is this the proposed semantics for WASM imports, for all of the exports to go into the default export, instead of `import * as wasmExports from "..."`?

Reply:
The standards seems to imply named exports. However we have problem collecting WASM imports and exports names since our current compiler does not support running some JS code. I don’t really know if there is a way I can directly get these info from Rust side other than either delegate this task to JS or using something like wasmer_runtime (which is too heavy ad it is a full runtime). Will investigate if there is some V8 APIs I could use instead...

Reply:
Actually I think I might be able to figure out export names with some extra tricks. Experimenting

Reply:
I mean there are V8 APIs for WASM, which we would hook into at a lower-level...

https://github.com/v8/v8/blob/3c8c1c9d0a7d51cb8634dba2d829bd7d99d99163/include/v8.h#L1562-L1572
https://github.com/v8/v8/blob/3c8c1c9d0a7d51cb8634dba2d829bd7d99d99163/include/v8.h#L4648-L4668
https://github.com/v8/v8/blob/3c8c1c9d0a7d51cb8634dba2d829bd7d99d99163/include/v8.h#L4670-L4729

Reply:
@ry I'm actually planning to do something similar to `TsCompiler` to delegate discovery of imports/exports through a separate worker, and construct a script based on the message posted back. These V8 APIs seems lacking access to import/export details...

Reply:
I'm not sure what you need exactly but I'm sure it's there somewhere. Check out GetModuleNamespace.

Reply:
@ry I believe that is from `v8::internal` that is not public? 
Anyways I am able to get named exports working now by having a JS worker compile and get imports/exports for me before I construct the static scripts.

Reply:
It's here https://github.com/v8/v8/blob/3c8c1c9d0a7d51cb8634dba2d829bd7d99d99163/include/v8.h#L1542